vuu-ui/packages/vuu-layout/src/layout-persistence/data.ts

```diff
@@ -0,0 +1,71 @@
+import { LayoutJSON } from "../layout-reducer";
+
+export const warningLayout: LayoutJSON = {
+  type: "View",
+  props: {
+    style: { height: "calc(100% - 6px)" },
+  },
+  children: [
+    {
+      props: {
+        className: "vuuShell-warningPlaceholder",
+      },
+      type: "Placeholder",
+    },
+  ],
+};
+
+export const defaultLayout: LayoutJSON = {
+  type: "Stack",
+  id: "main-tabs",
+  props: {
+    className: "vuuShell-mainTabs",
+    TabstripProps: {
+      allowAddTab: true,
+      allowRenameTab: true,
+      animateSelectionThumb: false,
+      location: "main-tab",
+    },
+    preserve: true,
+    active: 0,
+  },
+  children: [
+    {
```

Comment:
We need to remove these entries in the defaultLayout. They refer to a feature (FilterTable) which may not be available at runtime , depending on the app. 

The following will work anywhere, as the Placeholder will aways be available at runtime. 

```
export const defaultLayout: LayoutJSON = {
  type: "Stack",
  id: "main-tabs",
  props: {
    className: "vuuShell-mainTabs",
    TabstripProps: {
      allowAddTab: true,
      allowRenameTab: true,
      animateSelectionThumb: false,
      location: "main-tab",
    },
    preserve: true,
    active: 0,
  },
  children: [
    {
      props: {
        className: "vuuShell-Placeholder",
      },
      type: "Placeholder",
    },
  ],
};

```

Reply:
Are there any features which are guaranteed to be available at runtime? I liked having a pre-populated layout as opposed to an entirely empty one. It gives the user a better impression/expectation of what they can do.

Reply:
check with Steve tomorrow